test/functional/shada/shada_spec.lua

```diff
@@ -250,6 +250,21 @@ describe('ShaDa support code', function()
     end
   end)
 
+  it(
+    'allows writing and reading ShaDa files when shadafile=NONE but supplied with a filename',
+    function()
+      nvim_command('set shadafile=NONE')
+      eq(0, exc_exec('wshada ' .. shada_fname))
```

Comment:
`exc_exec` is deprecated, and not needed to check success.
```suggestion
      nvim_command('wshada ' .. shada_fname)
```